elastic/eui
# build: fix releases in monorepo project structure

## Summary

This fixes a few issues I encountered while releasing EUI after converting the repo to a monorepo structure. I recommend reviewing commit-by-commit to get more context.

As soon as we integrate a monorepo manager, we won't need the copied files, as they'll be automatically included from the root directory. For the time being, though, copying them to `packages/eui` is the simplest solution.

## QA

1. Checkout this PR locally
2. Go to `packages/eui`
3. Ensure you're logged out from npm as an extra precaution by running `npm logout`
4. [x] Run `npm run release -- --type=prerelease --dry-run` and confirm that it doesn't throw any errors and that the `Updated changelog` commit is created
5. [x] Run `npm run pack` and confirm `README.md`, `LICENSE.txt` and the `licenses` directory are included in the package
